unit_tests/testIOutil/testIOutil.f90

```diff
@@ -550,6 +550,30 @@ SUBROUTINE testIO_Strings()
       ASSERT_EQ(str(-5000.0_SDK),'-5.000000000000000E+03','str(SDK)')
       ASSERT_EQ(str(-5000.0_SDK,5),'-5.00000E+03','str(SDK,nDecimal)')
 
+      COMPONENT_TEST('getFloatFormat')
+      ASSERT_EQ(getFloatFormat(0.0_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-0.0_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(0.1_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-0.1_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(1.1_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-1.1_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(12.1_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-21.1_SRK),3_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(123.1_SRK),3_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-213.1_SRK),4_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(4213.1_SRK),4_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(0.01_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-0.01_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(0.123_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-0.123_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(1.1234_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-1.1342_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(12.12_SRK),2_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-21.12_SRK),3_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(123.12_SRK),3_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-213.12_SRK),4_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(4213.12_SRK),4_SIK,'zero')
```

Comment:
Also, you have `zero` as the message for all of these... I don't care to much about the message but using `zero` everywhere is wrong.  You could just use `float format` as the message in all of them.  The `ASSERT_EQ` will already print the information we care about